seed/views/v3/taxlots.py

```diff
@@ -10,31 +11,36 @@
 from rest_framework.renderers import JSONRenderer
 from seed.models import StatusLabel as Label
 from seed.utils.api import OrgMixin
-from seed.utils.api_schema import swagger_auto_schema_org_query_param
-from seed.utils.labels import _get_labels
+from seed.utils.api_schema import AutoSchemaHelper
+from seed.utils.labels import get_labels
 
 ErrorState = namedtuple('ErrorState', ['status_code', 'message'])
 
 
-class TaxlotLabelsViewSet(viewsets.ViewSet, OrgMixin):
+class TaxlotViewSet(viewsets.ViewSet, OrgMixin):
     renderer_classes = (JSONRenderer,)
     parser_classes = (JSONParser,)
     _organization = None
 
-    def get_queryset(self):
-        labels = Label.objects.filter(
-            super_organization=self.get_parent_org(self.request)
-        ).order_by("name").distinct()
-        return labels
-
-    @swagger_auto_schema_org_query_param
+    @swagger_auto_schema(
+        manual_parameters=[AutoSchemaHelper.query_org_id_field(required=False)],
+        request_body=AutoSchemaHelper.schema_factory(
+            {
+                'selected': ['integer'],
+            },
+            description='An array of taxlot ids to be returned'
+        )
+    )
     @action(detail=False, methods=['POST'])
     def labels(self, request):
         """
         Returns a list of all labels where the is_applied field
         in the response pertains to the labels applied to taxlot_view
         """
         inv_type = 'taxlot_view'
```

Comment:
What do you think about just putting 'taxlot_view' directly as an arg in line 46 and removing this variable declaration?